funcx/executors/high_throughput/funcx_worker.py

```diff
@@ -114,20 +114,23 @@ def start(self):
 
             try:
                 result = self.execute_task(msg)
-                logger.debug("Executed result: {}".format(result))
-                serialized_result = serialize_object(result)
+                serialized_result = self.serialize(result)
             except Exception as e:
                 logger.exception(f"Caught an exception {e}")
-                result_package = {'task_id': task_id, 'exception': serialize_object(
+                result_package = {'task_id': task_id, 'exception': self.serialize(
                     RemoteExceptionWrapper(*sys.exc_info()))}
             else:
                 logger.debug("Execution completed without exception")
-                result_package = {'task_id': task_id, 'result': serialized_result}
+                result_package = {'task_id': task_id,
+                                  'result': serialized_result}
 
             # TODO: Change this to serialize_object to match IX?
```

Comment:
I don't think this is a TODO anymore. Plan all along was to use our own serialization library, which we now do.

Reply:
Fixed.